packages/UI/src/components/ShareDialog/ShareDialog.tsx

```diff
@@ -0,0 +1,169 @@
+import {
+  Twitter,
+  Facebook,
+  Instagram,
+  Linkedin,
+  Link as LinkIcon,
+} from 'lucide-react';
+import {
+  FacebookShareButton,
+  TwitterShareButton,
+  LinkedinShareButton,
+  InstapaperShareButton,
+} from 'next-share';
+import { tv } from 'tailwind-variants';
+import Button from '../atoms/Button';
+import {
+  Dialog,
+  DialogTrigger,
+  DialogContent,
+  DialogHeader,
+  DialogFooter,
+  DialogTitle,
+} from './../Dialog';
+
+const ShareDialogStyles = tv(
+  {
+    slots: {
+      dialogHeader:
+        'flex h-6 w-[360px] items-center justify-center font-nunito-sans text-xl uppercase tracking-wider',
+      dialogFooter:
+        'flex w-[320px] flex-col items-center justify-end gap-4 font-nunito-sans',
+      dialogBody: 'flex w-[320px] items-center justify-evenly gap-4',
+      shareBtn: 'flex h-10 w-10 items-center justify-center rounded shadow-lg',
+    },

```

Comment:
I'm starting to feel that tailwind variants should be used for complex components rather than simpler ones with no different variants like this one. maybe it's better to move these class to the markup directly